vscode/microsoft-kiota/l10n/bundle.l10n.cs.json

```diff
@@ -38,5 +37,45 @@
   "Invalid URL, please check the documentation for the supported URLs": "Neplatná URL adresa, zkontrolujte dokumentaci pro podporované URL adresy",
   "No content found in the clipboard": "V schránce nebyl nalezen žádný obsah",
   "Invalid content found in the clipboard": "V schránce byl nalezen neplatný obsah",
-  "Select an API manifest key": "Vyberte klíč manifestu API"
+  "Select an API manifest key": "Vyberte klíč manifestu API",
+  "Edit Paths":"Upravit cesty",
+  "Re-generate": "Znovu vygenerovat",
+  "Add an API description": "Přidat popis API",
+  "Search or paste a path to an API description": "Hledat nebo vložit cestu k popisu API",
+  "Generate a plugin": "Generovat plugin",
+  "Generate an API manifest": "Generovat API manifest",
+  "What do you want to generate?": "Co chcete generovat? ",
+  "Select an option": "Vyberte možnost",
+  "class": "třída",
+  "namespace": "jmenný prostor",
+  "output directory": "výstupní adresář",
+  "language": "jazyk",
+  "Create a new API client": "Vytvořit nového API klienta",
+  "Create a new plugin": "Vytvořit nový plugin",
+  "Create a new manifest": "Vytvořit nový manifest",
+  "plugin name:": "název pluginu",
+  "Choose a name for the plugin": "Vyberte název pro plugin",
+  "Choose a plugin type": "Vyberte typ pluginu",
+  "manifest name": "název manifestu",
+  "Choose a name for the manifest": "Vyberte název pro manifest",
+  "Required": "Požadováno",
+  "Open installation instructions for kiota?": "Otevřít instalační pokyny pro kiota",
+  "Invalid generation type": "Neplatný typ generování",
+  "Yes, override it": "Ano, přepsat",
+  "Before adding a new API description, consider that your changes and current selection will be lost.": "Před přidáním nového popisu API zvažte, že vaše změny a aktuální výběr budou ztraceny.",
+  "Cancel": "Zrušit",
+  "Do you want to remove this API description?": "Chcete odstranit tento popis API?",
+  "Please save the workspace.json file before re-generation.": "Před generováním prosím uložte soubor workspace.json",
+  "OK": "OK",
+  "Generating manifest...": "Generování manifestu...",
+  "Generating plugin...": "Generování pluginu...",
+  "Re-generating client...": "Opětovné generování klienta...",
+  "Re-generating plugin...": "Opětovné generování pluginu...",
+  "Please migrate your API clients to Kiota workspace.":"",
```

Comment:
have we reached out to the translators to get these missing ones?

Reply:
Not yet, will do so